js/index.js

```diff
@@ -12,6 +12,67 @@ const selectRandom = (memberImgArr, n) => {
 
 const numOfMembers = 5;
 let memberSection = document.getElementById('members');
+
+let regexp = /android|iphone|kindle|ipad/i;
+isOpenInPhone(regexp);
+
+export function isOpenInPhone(type) {
+  let details = navigator.userAgent;
+  let isMobileDevice = type.test(details);
+
+  if (isMobileDevice) {
+    openDialog();
+    document.getElementById('okayBt').addEventListener('click', openApp);
+    document.getElementById('cancleBt').addEventListener('click', closeDialog);
```

Comment:
don't use shorthands for naming that are hard to understand, also there is typo in cancel button